src/lib/commandBarConfigs/modelingCommandConfig.ts

```diff
@@ -425,10 +425,10 @@ export const modelingMachineCommandConfig: StateMachineCommandSetConfig<
       },
       path: {
         inputType: 'selection',
-        selectionTypes: ['segment'],
+        selectionTypes: ['segment', 'helix'],
         required: true,
         multiple: false,
-        validation: sweepValidator,
+        // validation: sweepValidator,
```

Comment:
Discussion about removing these for v1:
 https://kittycadworkspace.slack.com/archives/C07A80B83FS/p1747315489085599

cc @franknoirot @nadr0

Reply:
Made a sep PR https://github.com/KittyCAD/modeling-app/pull/6969